<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="point">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Point</string>
   </property>
  </widget>
  <widget class="QLabel" name="press">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Press</string>
   </property>
  </widget>
  <widget class="QLabel" name="wheel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Wheel</string>
   </property>
  </widget>
  <widget class="QLabel" name="point_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="press_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wheel_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>590</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
